# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libofx/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- libofx-0.8.0/lib/tree.hh.orig	2004-04-09 08:51:45.000000000 +0200
+++ libofx-0.8.0/lib/tree.hh	2005-09-06 21:09:41.000000000 +0200
@@ -69,7 +69,7 @@
    }
 
 template <class T1>
-inline void kp::destructor(T1* p)
+inline void destructor(T1* p)
    {
    p->~T1();
    }
